### ISSUE TITLE
"Error decoding JSON data: Syntax error" message after 3.9.6 to 3.9.7 update

### ISSUE THREAD

**Issue by conseilgouz** [joomla/joomla-cms#25176]
### Steps to reproduce the issue
Update a french website where helpsite is help.joomla.fr

### Expected result
In #__users tables, params should contains "helpfile":""

### Actual result
In #__users tables, params contains ,, causing a JSON error

### System information (as much as possible)
Joomla 3.9.7

### Additional comments
File 3.9.7-2019-05-16.sql  contains incorrect reset information

**Comment by HLeithner** [comment]
@alikon could you confirm this?

**Comment by conseilgouz** [comment]
Correct command should be :
UPDATE `#__users` SET `params` = REPLACE(`params`, '"helpsite":"https:\\/\\/help.joomla.fr\\/index.php?option=com_help&keyref=Help{major}{minor}:{keyref}"', '"helpsite":""');

**Comment by HLeithner** [comment]
The new value have to be replaced for the next release and as workaround.

**Comment by conseilgouz** [comment]
French websites are going down one by one.....

**Comment by mbabker** [comment]
TBH, you would've been better off leaving the user params alone versus trying to remove it.  With the param gone from the UI, and the code not reading it, it would eventually cycle itself out as preferences get updated or just sit there and take up 50 bytes in a database field.

Now you've got corrupt JSON to deal with.

**Comment by HLeithner** [comment]
My suggestion to fix this would be.
```php
UPDATE #__users SET params = REPLACE(params, '",,"', '","');
```
Would this work without side effects?

**Comment by conseilgouz** [comment]
I tried 
UPDATE `#__users` SET `params` = REPLACE(`params`, ',,', ',"helpsite":"",');

It recreates helpsite param.

**Comment by HLeithner** [comment]
replacing  only ,, could lead to modify data of other fields

**Comment by richard67** [comment]
> 
> 
> I tried
> UPDATE `#__users` SET `params` = REPLACE(`params`, ',,', ',"helpsite":"",');
> 
> It recreates helpsite param.

There is no need to recreate the helpsite parameter, it is not used anymore in the code. It is only necessary to make Json valid again, i.e. remove the `,,`, but in a safe way.

**Comment by HLeithner** [comment]
I created a PR to fix this #25177

**Comment by HLeithner** [comment]
normally the helpsite is between 2 other parameters I don't think that it's possible that the ,, are at the beginning or the end...

**Comment by richard67** [comment]
@HLeithner I think the same, but am not 100% sure. But I guess you wanted to comment that in your PR.

**Comment by alikon** [comment]
@pmleconte please test #25177

**Comment by richard67** [comment]
@pmleconte Fixed with new release 3.9.8.

**Comment by richard67** [comment]
@pmleconte And thanks for reporting it so fast.

**Comment by zero-24** [comment]
closing as this has been patched by 3.9.8 thanks everyone involved.
